chromium/background-scripts/update.js

```diff
@@ -198,11 +201,15 @@ async function applyStoredRulesets(rulesets_obj) {
 
   const channel_results = (await Promise.all(rulesets_promises)).filter(isNotUndefined);
 
-  if(channel_results.length > 0) {
-    for(let channel_result of channel_results) {
-      rulesets_obj.addFromJson(channel_result.json.rulesets, channel_result.scope);
+  let replaces = false;
+  for(let channel_result of channel_results) {
+    if(channel_result.replaces == true) {
```

Comment:
Is it safe to assume `channel_result.replaces == true` implies the content of `channel_result.json.rulesets` originates from EFF? My understanding is that if an evil `update_channel` crafted the response such that `channel_result.replaces` is `true`,  the bundled rules from EFF might never apply.

Reply:
That is the assumption - there is no way in the update channels UX to add an update channel that has the replacement flag set to true, and I don't plan on exposing that to the UX either.  This means that adding an update channel will not result in your default rulesets being overwritten.

Reply:
I see your point and I believe this is a reasonable assumption. (though I guess this should be explicitly documented somewhere, possibly in this or later PR).